protocol/x/clob/keeper/liquidations.go

```diff
@@ -92,23 +98,14 @@ func (k Keeper) LiquidateSubaccountsAgainstOrderbook(
 		optimisticallyFilledQuantums, _, err := k.PlacePerpetualLiquidation(ctx, *liquidationOrder)
 		if err != nil {
 			k.Logger(ctx).Error(
-				fmt.Sprintf(
-					"Failed to liquidate subaccount. Liquidation Order: (%+v). Err: %v",
-					liquidationOrder,
-					err,
-				),
+				"Failed to liquidate subaccount",
+				"liquidationOrder", liquidationOrder,
+				"error", err,
 			)
 			return err
 		}
 
-		// Keep a count of partially and fully filled liquidations for this block.
```

Comment:
Do you think it would be helpful to stat out the total amount of quantums successfully deleveraged, per market, in each block? Asking because I think statting out each individual value could be hard to aggregrate in datadog since the value would likely fluctuate a lot between each deleveraging event

Reply:
hmm how would you make this per block on datadog without adding block height as labels?

Reply:
If you sum up the total amount of quantums deleveraged and emit it as one stat, wouldn't you know that's for one block?

Agree you wouldn't know the exact block height though